game.js

```diff
@@ -28,7 +32,9 @@ const quitGame = () => {
  * @returns {undefined}
  */
 const gameLoop = () => {
-
+  console.log("I have a random number in mind")
+  console.log("It's between 1 and 1000")
+  console.log("You have 10 guesses total")
```

Comment:
Missing the user stories here as outlined by the ReadMe. Needs to be able to play the game. Please revisit.